pkg/operator/k8sutil/node.go

```diff
@@ -246,3 +247,41 @@ func NodeIsInRookNodeList(targetNodeName string, rookNodes []rookalpha.Node) boo
 	}
 	return false
 }
+
+// AddNodeAffinity will return a v1.NodeAffinity object
+func AddNodeAffinity(nodeAffinity string) *v1.NodeAffinity {
```

Comment:
If there is an error parsing the desired affinity, what is the expected behavior? Will the affinity be ignored and skipped? Or will the operator return an error and prevent continuing until it's fixed? I think i'd vote for ignoring a bad affinity and writing the error to the log.

Reply:
Agreed. Will take care of this.